Makefile

```diff
@@ -175,5 +188,37 @@ test-e2e: generate fmt vet manifests
 	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test $(TEST_E2E_PKGS) -coverprofile cover.out -ginkgo.v -ginkgo.progress -v
 
+.PHONY: bundle-validate
 bundle-validate:
 	$(OPERATOR_SDK) bundle validate ./bundle
+
+.PHONY: bundle-validate-image
+bundle-validate-image:
+	$(OPERATOR_SDK) bundle validate $(BUNDLE_IMG)
+
+.PHONY: bundle-custom-updates
+bundle-custom-updates: BUNDLE_PREFIX=dev$(CURRENT_DATE)
+bundle-custom-updates:
+ifndef YQ
+	$(error "yq is not available please install: https://github.com/mikefarah/yq/releases/latest")
+endif
+	@echo "Update metadata to avoid collision with existing APIcast Operator official public operators catalog entries"
```

Comment:
Probably a good idea to print bundle prefix value too, as the changes are automatically reverted when running `bundle-custom-build` and it will be useful to know how it's been created

Reply:
Adding it. 

Anyway, when you run last step, `make bundle-run BUNDLE_IMG=quay.io/myorg/apicast-operator-bundles:myversiontag`, the command will show the prefix:
```
$ make bundle-run BUNDLE_IMG=quay.io/eastizle/bundle-catalogs:apicast-testing-02
operator-sdk run bundle --namespace eguzki quay.io/eastizle/bundle-catalogs:apicast-testing-02
INFO[0015] Successfully created registry pod: quay-io-eastizle-bundle-catalogs-apicast-testing-02 
INFO[0015] Created CatalogSource: dev1605117482-apicast-operator-catalog 
INFO[0016] Created Subscription: dev1605117482-apicast-operator-v0-0-1-sub 
INFO[0020] Approved InstallPlan install-7vpsh for the Subscription: dev1605117482-apicast-operator-v0-0-1-sub 
INFO[0021] Waiting for ClusterServiceVersion "eguzki/dev1605117482-apicast-operator.v0.0.1" to reach 'Succeeded' phase 
INFO[0021]   Waiting for ClusterServiceVersion "eguzki/dev1605117482-apicast-operator.v0.0.1" to appear 
INFO[0036]   Found ClusterServiceVersion "eguzki/dev1605117482-apicast-operator.v0.0.1" phase: Pending 
INFO[0038]   Found ClusterServiceVersion "eguzki/dev1605117482-apicast-operator.v0.0.1" phase: Installing 
INFO[0042]   Found ClusterServiceVersion "eguzki/dev1605117482-apicast-operator.v0.0.1" phase: Succeeded 
INFO[0042] OLM has successfully installed "dev1605117482-apicast-operator.v0.0.1"
```

Note that for this example, the prefix has been: `dev1605117482`